### PR TITLE
[contractkit]Cleanup: reference internal files directly

### PR DESCRIPTION
**Pull request by ashishb**
### Description

Earlier, the internal files were being referenced via
`@celo/contractkit` import path.
Now, reference them directly.

### Tested

`packages/contractkit$ yarn test`
`packages/mobile $ yarn run dev` - Verify that the mobile app works.

### CODE REVIEW

**Review by jmrossy — APPROVED** [review]
I'm a bit surprised but pleased that this doesn't break the mobile app bundling :)

File: packages/contractkit/src/contract-utils-v2.ts [hunk 1]
```diff
@@ -1,12 +1,12 @@
-import { Exchange } from '@celo/contractkit/types/Exchange'
-import { GasPriceMinimum as GasPriceMinimumType } from '@celo/contractkit/types/GasPriceMinimum'
-import { GoldToken as GoldTokenType } from '@celo/contractkit/types/GoldToken'
-import { StableToken as StableTokenType } from '@celo/contractkit/types/StableToken'
 import { CURRENCY_ENUM as Tokens } from '@celo/utils'
 import BigNumber from 'bignumber.js'
 import Web3 from 'web3'
 import { TransactionObject } from 'web3/eth/types'
 import { TransactionReceipt } from 'web3/types'
+import { Exchange } from '../types/Exchange'
```
Comment (yerdua) [inline packages/contractkit/src/contract-utils-v2.ts:6]: All the other imports from `../types` follow the pattern of `{ Foo as FooType }`. Why does this one break from that pattern?
(I see it was already like this before your change, so this is possibly out of scope)

**Review by yerdua — APPROVED** [review]
LGTM!

However, this highlights an inconsistency in when we use `import { Foo }` vs `import { Foo as FooType }`. If this is an intentional difference, I'd love to understand it. If it's unintentional, it's worth making a decision and making the code be consistent. This is a pre-existing issue though, and out of scope of this cleanup PR, so it's not a blocker.

File: packages/contractkit/src/contract-utils-v2.ts (hunk 1 above)
Comment (ashishb) [reply packages/contractkit/src/contract-utils-v2.ts:6]: I agree we should make this consistent in the contractkit everywhere. I'll send a follow-on PR.

**Pull request merged**